velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemTest.cpp

```diff
@@ -113,7 +112,17 @@ TEST_F(S3FileSystemTest, missingFile) {
   auto hiveConfig = minioServer_->hiveConfig();
   filesystems::S3FileSystem s3fs(hiveConfig);
   s3fs.initializeClient();
-  EXPECT_THROW(s3fs.openFileForRead(s3File), VeloxException);
+  try {
+    s3fs.openFileForRead(s3File);
```

Comment:
just testing for `VeloxException` is insufficient. Improved the test to capture the right message.